### ISSUE TITLE
docker image won't build:./lint.sh: line 8: prospector: command not found

### ISSUE THREAD

**Issue by jeffreybreen** [databricks/databricks-cli#166]
Dockerfile in master fails to build image with error `./lint.sh: line 8: prospector: command not found`

```
$ docker build -t databricks-cli .
Sending build context to Docker daemon    834kB
Step 1/5 : FROM python:2.7
 ---> 17c0fe4e76a5
Step 2/5 : WORKDIR /usr/src/databricks-cli
 ---> Using cache
 ---> b5504b00245b
Step 3/5 : COPY . .
 ---> Using cache
 ---> a11d261149fd
Step 4/5 : RUN pip install --upgrade pip &&     pip install -r dev-requirements.txt &&     pip list &&     ./lint.sh &&     pip install . &&     pytest tests
 ---> Running in b6cca86af096
Requirement already up-to-date: pip in /usr/local/lib/python2.7/site-packages (18.0)
Collecting tox==2.9.1 (from -r dev-requirements.txt (line 1))
  Downloading https://files.pythonhosted.org/packages/1d/4e/20c679f8c5948f7c48591fde33d442e716af66a31a88f5791850a75041eb/tox-2.9.1-py2.py3-none-any.whl (73kB)
Requirement already satisfied: virtualenv>=1.11.2; python_version != "3.2" in /usr/local/lib/python2.7/site-packages (from tox==2.9.1->-r dev-requirements.txt (line 1)) (16.0.0)
Collecting py>=1.4.17 (from tox==2.9.1->-r dev-requirements.txt (line 1))
  Downloading https://files.pythonhosted.org/packages/f3/bd/83369ff2dee18f22f27d16b78dd651e8939825af5f8b0b83c38729069962/py-1.5.4-py2.py3-none-any.whl (83kB)
Collecting six (from tox==2.9.1->-r dev-requirements.txt (line 1))
  Downloading https://files.pythonhosted.org/packages/67/4b/141a581104b1f6397bfa78ac9d43d8ad29a7ca43ea90a2d863fe3056e86a/six-1.11.0-py2.py3-none-any.whl
Collecting pluggy<1.0,>=0.3.0 (from tox==2.9.1->-r dev-requirements.txt (line 1))
  Downloading https://files.pythonhosted.org/packages/f5/f1/5a93c118663896d83f7bcbfb7f657ce1d0c0d617e6b4a443a53abcc658ca/pluggy-0.7.1-py2.py3-none-any.whl
Installing collected packages: py, six, pluggy, tox
Successfully installed pluggy-0.7.1 py-1.5.4 six-1.11.0 tox-2.9.1
Package    Version
---------- -------
pip        18.0
pluggy     0.7.1
py         1.5.4
setuptools 40.0.0
six        1.11.0
tox        2.9.1
virtualenv 16.0.0
wheel      0.31.1
./lint.sh: line 8: prospector: command not found
The command '/bin/sh -c pip install --upgrade pip &&     pip install -r dev-requirements.txt &&     pip list &&     ./lint.sh &&     pip install . &&     pytest tests' returned a non-zero code: 127
```

**Comment by jeffreybreen** [comment]
Proposed fixes in [Pull Request #167](https://github.com/databricks/databricks-cli/pull/167)

```
$ docker build -t databricks-cli .
Sending build context to Docker daemon  862.2kB
Step 1/5 : FROM python:2.7
 ---> 17c0fe4e76a5
Step 2/5 : WORKDIR /usr/src/databricks-cli
 ---> Using cache
 ---> 7ad7f229d315
Step 3/5 : COPY . .
 ---> d525af03a50b
Step 4/5 : RUN pip install --upgrade --no-cache-dir pip &&       pip install --no-cache-dir             -r dev-requirements.txt            -r tox-requirements.txt &&     pip list &&     ./lint.sh &&     pip install --no-cache-dir . &&     pytest tests  &&     pip uninstall -y         -r tox-requirements.txt
 ---> Running in f8c93896b32e
Requirement already up-to-date: pip in /usr/local/lib/python2.7/site-packages (18.0)
Collecting tox==3.2.1 (from -r dev-requirements.txt (line 1))
  Downloading https://files.pythonhosted.org/packages/df/53/13660f04ef09ece4aefcce6b8f79c1586fc34dee1cbedd7c153e02f93489/tox-3.2.1-py2.py3-none-any.whl (62kB)
Collecting click==6.7 (from -r dev-requirements.txt (line 2))
  Downloading https://files.pythonhosted.org/packages/34/c1/8806f99713ddb993c5366c362b2f908f18269f8d792aff1abfd700775a77/click-6.7-py2.py3-none-any.whl (71kB)
Collecting tabulate==0.8.2 (from -r dev-requirements.txt (line 3))
  Downloading https://files.pythonhosted.org/packages/12/c2/11d6845db5edf1295bc08b2f488cf5937806586afe42936c3f34c097ebdc/tabulate-0.8.2.tar.gz (45kB)
Collecting prospector[with_pyroma]==1.1.1 (from -r tox-requirements.txt (line 2))
  Downloading https://files.pythonhosted.org/packages/8a/94/d1643eb5460f3accc4609e2fe14ffe9e6dab2b1014b047fccab68b3ee5cb/prospector-1.1.1-py2.py3-none-any.whl (69kB)
Collecting pylint==1.9.3 (from -r tox-requirements.txt (line 3))
  Downloading https://files.pythonhosted.org/packages/40/5f/2ba2603512cbf48b2d5fa9c3ed494d61bef84fcf2f7a03e602cd15929727/pylint-1.9.3-py2.py3-none-any.whl (694kB)
Collecting pep8-naming==0.7.0 (from -r tox-requirements.txt (line 4))
  Downloading https://files.pythonhosted.org/packages/39/bb/a34544c789e7e5458ed2db6cbd1c8e227bb01e4ce03a0b15ec4ec93e486d/pep8_naming-0.7.0-py2.py3-none-any.whl
Collecting pytest==3.7.1 (from -r tox-requirements.txt (line 5))
  Downloading https://files.pythonhosted.org/packages/d2/86/7b9513da923b94e48c2cf013ae4eae8184a36ebeb7fe27d386bc3db4f56f/pytest-3.7.1-py2.py3-none-any.whl (202kB)
Collecting mock==2.0.0 (from -r tox-requirements.txt (line 6))
  Downloading https://files.pythonhosted.org/packages/e6/35/f187bdf23be87092bd0f1200d43d23076cee4d0dec109f195173fd3ebc79/mock-2.0.0-py2.py3-none-any.whl (56kB)
Collecting decorator==4.3.0 (from -r tox-requirements.txt (line 7))
  Downloading https://files.pythonhosted.org/packages/bc/bb/a24838832ba35baf52f32ab1a49b906b5f82fb7c76b2f6a7e35e140bac30/decorator-4.3.0-py2.py3-none-any.whl
Collecting rstcheck==3.3 (from -r tox-requirements.txt (line 8))
  Downloading https://files.pythonhosted.org/packages/ea/6e/c7a910dce2cafa17942b456cdbb5cf637a87a549e491d794b424b90e2bb1/rstcheck-3.3.tar.gz
Collecting pytest-cov (from -r tox-requirements.txt (line 9))
  Downloading https://files.pythonhosted.org/packages/30/7d/7f6a78ae44a1248ee28cc777586c18b28a1df903470e5d34a6e25712b8aa/pytest_cov-2.5.1-py2.py3-none-any.whl
Collecting codecov (from -r tox-requirements.txt (line 10))
  Downloading https://files.pythonhosted.org/packages/8b/28/4c1950a61c3c5786f0f34d643d0d28ec832433c9a7c0bd157690d4eb1d5f/codecov-2.0.15-py2.py3-none-any.whl
Collecting pluggy<1,>=0.3.0 (from tox==3.2.1->-r dev-requirements.txt (line 1))
  Downloading https://files.pythonhosted.org/packages/f5/f1/5a93c118663896d83f7bcbfb7f657ce1d0c0d617e6b4a443a53abcc658ca/pluggy-0.7.1-py2.py3-none-any.whl
Collecting py<2,>=1.4.17 (from tox==3.2.1->-r dev-requirements.txt (line 1))
  Downloading https://files.pythonhosted.org/packages/f3/bd/83369ff2dee18f22f27d16b78dd651e8939825af5f8b0b83c38729069962/py-1.5.4-py2.py3-none-any.whl (83kB)
Requirement already satisfied: virtualenv>=1.11.2 in /usr/local/lib/python2.7/site-packages (from tox==3.2.1->-r dev-requirements.txt (line 1)) (16.0.0)
Requirement already satisfied: setuptools>=30.0.0 in /usr/local/lib/python2.7/site-packages (from tox==3.2.1->-r dev-requirements.txt (line 1)) (40.0.0)
Collecting six<2,>=1.0.0 (from tox==3.2.1->-r dev-requirements.txt (line 1))
  Downloading https://files.pythonhosted.org/packages/67/4b/141a581104b1f6397bfa78ac9d43d8ad29a7ca43ea90a2d863fe3056e86a/six-1.11.0-py2.py3-none-any.whl
Collecting dodgy>=0.1.9 (from prospector[with_pyroma]==1.1.1->-r tox-requirements.txt (line 2))
  Downloading https://files.pythonhosted.org/packages/6a/ab/752d8ec6b21dfaa9e24e93976b0ef922fca990e80d88627c0dd8eed38f63/dodgy-0.1.9.tar.gz
Collecting setoptconf>=0.2.0 (from prospector[with_pyroma]==1.1.1->-r tox-requirements.txt (line 2))
  Downloading https://files.pythonhosted.org/packages/6b/5a/9186be8db0804d808a5a82a9b7e47e6260832a5e5b5ec531a1de9dd49076/setoptconf-0.2.0.tar.gz
Collecting pycodestyle<2.4.0,>=2.0.0 (from prospector[with_pyroma]==1.1.1->-r tox-requirements.txt (line 2))
  Downloading https://files.pythonhosted.org/packages/e4/81/78fe51eb4038d1388b7217dd63770b0f428370207125047312886c923b26/pycodestyle-2.3.1-py2.py3-none-any.whl (45kB)
Collecting requirements-detector>=0.6 (from prospector[with_pyroma]==1.1.1->-r tox-requirements.txt (line 2))
  Downloading https://files.pythonhosted.org/packages/3a/99/fb4251c8a71e9e58fadfc21a71ce9a77b41684c58ea0f8d4ac7895370a96/requirements-detector-0.6.tar.gz
Collecting mccabe>=0.5.0 (from prospector[with_pyroma]==1.1.1->-r tox-requirements.txt (line 2))
  Downloading https://files.pythonhosted.org/packages/87/89/479dc97e18549e21354893e4ee4ef36db1d237534982482c3681ee6e7b57/mccabe-0.6.1-py2.py3-none-any.whl
Collecting pydocstyle>=2.0.0 (from prospector[with_pyroma]==1.1.1->-r tox-requirements.txt (line 2))
  Downloading https://files.pythonhosted.org/packages/4e/20/c710bd30d53c2786d1d9980a5f66598fb3e7348699858f40def7d625dedb/pydocstyle-2.1.1-py2-none-any.whl
Collecting pyyaml (from prospector[with_pyroma]==1.1.1->-r tox-requirements.txt (line 2))
  Downloading https://files.pythonhosted.org/packages/9e/a3/1d13970c3f36777c583f136c136f804d70f500168edc1edea6daa7200769/PyYAML-3.13.tar.gz (270kB)
Collecting pylint-plugin-utils>=0.2.6 (from prospector[with_pyroma]==1.1.1->-r tox-requirements.txt (line 2))
  Downloading https://files.pythonhosted.org/packages/b1/c1/75a481b103fecf8065300681c460dbed80105098c5d63f995ab0949fbfac/pylint-plugin-utils-0.4.tar.gz
Collecting pyflakes<2.0.0,>=0.8.1 (from prospector[with_pyroma]==1.1.1->-r tox-requirements.txt (line 2))
  Downloading https://files.pythonhosted.org/packages/d7/40/733bcc64da3161ae4122c11e88269f276358ca29335468005cb0ee538665/pyflakes-1.6.0-py2.py3-none-any.whl (227kB)
Collecting pyroma>=2.3; extra == "with_pyroma" (from prospector[with_pyroma]==1.1.1->-r tox-requirements.txt (line 2))
  Downloading https://files.pythonhosted.org/packages/ac/27/bd35e2745c676579fb9df6f4011ab7bef6231033360ba6f9f0d6abadebb7/pyroma-2.3.1.tar.gz (346kB)
Collecting isort>=4.2.5 (from pylint==1.9.3->-r tox-requirements.txt (line 3))
  Downloading https://files.pythonhosted.org/packages/41/d8/a945da414f2adc1d9e2f7d6e7445b27f2be42766879062a2e63616ad4199/isort-4.3.4-py2-none-any.whl (45kB)
Collecting singledispatch; python_version < "3.4" (from pylint==1.9.3->-r tox-requirements.txt (line 3))
  Downloading https://files.pythonhosted.org/packages/c5/10/369f50bcd4621b263927b0a1519987a04383d4a98fb10438042ad410cf88/singledispatch-3.4.0.3-py2.py3-none-any.whl
Collecting astroid<2.0,>=1.6 (from pylint==1.9.3->-r tox-requirements.txt (line 3))
  Downloading https://files.pythonhosted.org/packages/0e/9b/18b08991c8c6aaa827faf394f4468b8fee41db1f73aa5157f9f5fb2e69c3/astroid-1.6.5-py2.py3-none-any.whl (293kB)
Collecting configparser; python_version == "2.7" (from pylint==1.9.3->-r tox-requirements.txt (line 3))
  Downloading https://files.pythonhosted.org/packages/7c/69/c2ce7e91c89dc073eb1aa74c0621c3eefbffe8216b3f9af9d3885265c01c/configparser-3.5.0.tar.gz
Collecting backports.functools-lru-cache; python_version == "2.7" (from pylint==1.9.3->-r tox-requirements.txt (line 3))
  Downloading https://files.pythonhosted.org/packages/03/8e/2424c0e65c4a066e28f539364deee49b6451f8fcd4f718fefa50cc3dcf48/backports.functools_lru_cache-1.5-py2.py3-none-any.whl
Collecting flake8-polyfill<2,>=1.0.2 (from pep8-naming==0.7.0->-r tox-requirements.txt (line 4))
  Downloading https://files.pythonhosted.org/packages/86/b5/a43fed6fd0193585d17d6faa7b85317d4461f694aaed546098c69f856579/flake8_polyfill-1.0.2-py2.py3-none-any.whl
Collecting atomicwrites>=1.0 (from pytest==3.7.1->-r tox-requirements.txt (line 5))
  Downloading https://files.pythonhosted.org/packages/0a/e8/cd6375e7a59664eeea9e1c77a766eeac0fc3083bb958c2b41ec46b95f29c/atomicwrites-1.1.5-py2.py3-none-any.whl
Collecting pathlib2>=2.2.0; python_version < "3.6" (from pytest==3.7.1->-r tox-requirements.txt (line 5))
  Downloading https://files.pythonhosted.org/packages/66/a7/9f8d84f31728d78beade9b1271ccbfb290c41c1e4dc13dbd4997ad594dcd/pathlib2-2.3.2-py2.py3-none-any.whl
Collecting funcsigs; python_version < "3.0" (from pytest==3.7.1->-r tox-requirements.txt (line 5))
  Downloading https://files.pythonhosted.org/packages/69/cb/f5be453359271714c01b9bd06126eaf2e368f1fddfff30818754b5ac2328/funcsigs-1.0.2-py2.py3-none-any.whl
Collecting more-itertools>=4.0.0 (from pytest==3.7.1->-r tox-requirements.txt (line 5))
  Downloading https://files.pythonhosted.org/packages/fb/d3/77f337876600747ae307ea775ff264c5304a691941cd347382c7932c60ad/more_itertools-4.3.0-py2-none-any.whl (48kB)
Collecting attrs>=17.4.0 (from pytest==3.7.1->-r tox-requirements.txt (line 5))
  Downloading https://files.pythonhosted.org/packages/41/59/cedf87e91ed541be7957c501a92102f9cc6363c623a7666d69d51c78ac5b/attrs-18.1.0-py2.py3-none-any.whl
Collecting pbr>=0.11 (from mock==2.0.0->-r tox-requirements.txt (line 6))
  Downloading https://files.pythonhosted.org/packages/69/1c/98cba002ed975a91a0294863d9c774cc0ebe38e05bbb65e83314550b1677/pbr-4.2.0-py2.py3-none-any.whl (100kB)
Collecting docutils>=0.7 (from rstcheck==3.3->-r tox-requirements.txt (line 8))
  Downloading https://files.pythonhosted.org/packages/50/09/c53398e0005b11f7ffb27b7aa720c617aba53be4fb4f4f3f06b9b5c60f28/docutils-0.14-py2-none-any.whl (543kB)
Collecting coverage>=3.7.1 (from pytest-cov->-r tox-requirements.txt (line 9))
  Downloading https://files.pythonhosted.org/packages/31/22/724704999cf994b590db9421b5ef1b5122ea485d67d56f9ecf34f2f721ad/coverage-4.5.1-cp27-cp27mu-manylinux1_x86_64.whl (199kB)
Collecting requests>=2.7.9 (from codecov->-r tox-requirements.txt (line 10))
  Downloading https://files.pythonhosted.org/packages/65/47/7e02164a2a3db50ed6d8a6ab1d6d60b69c4c3fdf57a284257925dfc12bda/requests-2.19.1-py2.py3-none-any.whl (91kB)
Collecting snowballstemmer (from pydocstyle>=2.0.0->prospector[with_pyroma]==1.1.1->-r tox-requirements.txt (line 2))
  Downloading https://files.pythonhosted.org/packages/d4/6c/8a935e2c7b54a37714656d753e4187ee0631988184ed50c0cf6476858566/snowballstemmer-1.2.1-py2.py3-none-any.whl (64kB)
Collecting futures (from isort>=4.2.5->pylint==1.9.3->-r tox-requirements.txt (line 3))
  Downloading https://files.pythonhosted.org/packages/2d/99/b2c4e9d5a30f6471e410a146232b4118e697fa3ffc06d6a65efde84debd0/futures-3.2.0-py2-none-any.whl
Collecting enum34>=1.1.3; python_version < "3.4" (from astroid<2.0,>=1.6->pylint==1.9.3->-r tox-requirements.txt (line 3))
  Downloading https://files.pythonhosted.org/packages/c5/db/e56e6b4bbac7c4a06de1c50de6fe1ef3810018ae11732a50f15f62c7d050/enum34-1.1.6-py2-none-any.whl
Collecting wrapt (from astroid<2.0,>=1.6->pylint==1.9.3->-r tox-requirements.txt (line 3))
  Downloading https://files.pythonhosted.org/packages/a0/47/66897906448185fcb77fc3c2b1bc20ed0ecca81a0f2f88eda3fc5a34fc3d/wrapt-1.10.11.tar.gz
Collecting lazy-object-proxy (from astroid<2.0,>=1.6->pylint==1.9.3->-r tox-requirements.txt (line 3))
  Downloading https://files.pythonhosted.org/packages/52/7e/f0f570ba363e15251bb9fd452257ec2aff91be0187a08a893afbd8ae225f/lazy_object_proxy-1.3.1-cp27-cp27mu-manylinux1_x86_64.whl (56kB)
Collecting flake8 (from flake8-polyfill<2,>=1.0.2->pep8-naming==0.7.0->-r tox-requirements.txt (line 4))
  Downloading https://files.pythonhosted.org/packages/b9/dc/14e9d94c770b8c4ef584e906c7583e74864786a58d47de101f2767d50c0b/flake8-3.5.0-py2.py3-none-any.whl (69kB)
Collecting scandir; python_version < "3.5" (from pathlib2>=2.2.0; python_version < "3.6"->pytest==3.7.1->-r tox-requirements.txt (line 5))
  Downloading https://files.pythonhosted.org/packages/16/2a/557af1181e6b4e30254d5a6163b18f5053791ca66e251e77ab08887e8fe3/scandir-1.9.0.tar.gz
Collecting chardet<3.1.0,>=3.0.2 (from requests>=2.7.9->codecov->-r tox-requirements.txt (line 10))
  Downloading https://files.pythonhosted.org/packages/bc/a9/01ffebfb562e4274b6487b4bb1ddec7ca55ec7510b22e4c51f14098443b8/chardet-3.0.4-py2.py3-none-any.whl (133kB)
Collecting urllib3<1.24,>=1.21.1 (from requests>=2.7.9->codecov->-r tox-requirements.txt (line 10))
  Downloading https://files.pythonhosted.org/packages/bd/c9/6fdd990019071a4a32a5e7cb78a1d92c53851ef4f56f62a3486e6a7d8ffb/urllib3-1.23-py2.py3-none-any.whl (133kB)
Collecting certifi>=2017.4.17 (from requests>=2.7.9->codecov->-r tox-requirements.txt (line 10))
  Downloading https://files.pythonhosted.org/packages/16/1f/50d729c104b21c1042aa51560da6141d1cab476ba7015d92b2111c8db841/certifi-2018.8.13-py2.py3-none-any.whl (146kB)
Collecting idna<2.8,>=2.5 (from requests>=2.7.9->codecov->-r tox-requirements.txt (line 10))
  Downloading https://files.pythonhosted.org/packages/4b/2a/0276479a4b3caeb8a8c1af2f8e4355746a97fab05a372e4a2c6a6b876165/idna-2.7-py2.py3-none-any.whl (58kB)
prospector 1.1.1 has requirement pylint>=2, but you'll have pylint 1.9.3 which is incompatible.
Installing collected packages: pluggy, py, six, tox, click, tabulate, dodgy, setoptconf, pycodestyle, enum34, wrapt, singledispatch, backports.functools-lru-cache, lazy-object-proxy, astroid, requirements-detector, configparser, mccabe, pyflakes, flake8, flake8-polyfill, pep8-naming, snowballstemmer, pydocstyle, pyyaml, futures, isort, pylint, pylint-plugin-utils, docutils, pyroma, prospector, atomicwrites, scandir, pathlib2, funcsigs, more-itertools, attrs, pytest, pbr, mock, decorator, rstcheck, coverage, pytest-cov, chardet, urllib3, certifi, idna, requests, codecov
  Running setup.py install for tabulate: started
    Running setup.py install for tabulate: finished with status 'done'
  Running setup.py install for dodgy: started
    Running setup.py install for dodgy: finished with status 'done'
  Running setup.py install for setoptconf: started
    Running setup.py install for setoptconf: finished with status 'done'
  Running setup.py install for wrapt: started
    Running setup.py install for wrapt: finished with status 'done'
  Running setup.py install for requirements-detector: started
    Running setup.py install for requirements-detector: finished with status 'done'
  Running setup.py install for configparser: started
    Running setup.py install for configparser: finished with status 'done'
  Running setup.py install for pyyaml: started
    Running setup.py install for pyyaml: finished with status 'done'
  Running setup.py install for pylint-plugin-utils: started
    Running setup.py install for pylint-plugin-utils: finished with status 'done'
  Running setup.py install for pyroma: started
    Running setup.py install for pyroma: finished with status 'done'
  Running setup.py install for scandir: started
    Running setup.py install for scandir: finished with status 'done'
  Running setup.py install for rstcheck: started
    Running setup.py install for rstcheck: finished with status 'done'
Successfully installed astroid-1.6.5 atomicwrites-1.1.5 attrs-18.1.0 backports.functools-lru-cache-1.5 certifi-2018.8.13 chardet-3.0.4 click-6.7 codecov-2.0.15 configparser-3.5.0 coverage-4.5.1 decorator-4.3.0 docutils-0.14 dodgy-0.1.9 enum34-1.1.6 flake8-3.5.0 flake8-polyfill-1.0.2 funcsigs-1.0.2 futures-3.2.0 idna-2.7 isort-4.3.4 lazy-object-proxy-1.3.1 mccabe-0.6.1 mock-2.0.0 more-itertools-4.3.0 pathlib2-2.3.2 pbr-4.2.0 pep8-naming-0.7.0 pluggy-0.7.1 prospector-1.1.1 py-1.5.4 pycodestyle-2.3.1 pydocstyle-2.1.1 pyflakes-1.6.0 pylint-1.9.3 pylint-plugin-utils-0.4 pyroma-2.3.1 pytest-3.7.1 pytest-cov-2.5.1 pyyaml-3.13 requests-2.19.1 requirements-detector-0.6 rstcheck-3.3 scandir-1.9.0 setoptconf-0.2.0 singledispatch-3.4.0.3 six-1.11.0 snowballstemmer-1.2.1 tabulate-0.8.2 tox-3.2.1 urllib3-1.23 wrapt-1.10.11
Package                       Version  
----------------------------- ---------
astroid                       1.6.5    
atomicwrites                  1.1.5    
attrs                         18.1.0   
backports.functools-lru-cache 1.5      
certifi                       2018.8.13
chardet                       3.0.4    
click                         6.7      
codecov                       2.0.15   
configparser                  3.5.0    
coverage                      4.5.1    
decorator                     4.3.0    
docutils                      0.14     
dodgy                         0.1.9    
enum34                        1.1.6    
flake8                        3.5.0    
flake8-polyfill               1.0.2    
funcsigs                      1.0.2    
futures                       3.2.0    
idna                          2.7      
isort                         4.3.4    
lazy-object-proxy             1.3.1    
mccabe                        0.6.1    
mock                          2.0.0    
more-itertools                4.3.0    
pathlib2                      2.3.2    
pbr                           4.2.0    
pep8-naming                   0.7.0    
pip                           18.0     
pluggy                        0.7.1    
prospector                    1.1.1    
py                            1.5.4    
pycodestyle                   2.3.1    
pydocstyle                    2.1.1    
pyflakes                      1.6.0    
pylint                        1.9.3    
pylint-plugin-utils           0.4      
pyroma                        2.3.1    
pytest                        3.7.1    
pytest-cov                    2.5.1    
PyYAML                        3.13     
requests                      2.19.1   
requirements-detector         0.6      
rstcheck                      3.3      
scandir                       1.9.0    
setoptconf                    0.2.0    
setuptools                    40.0.0   
singledispatch                3.4.0.3  
six                           1.11.0   
snowballstemmer               1.2.1    
tabulate                      0.8.2    
tox                           3.2.1    
urllib3                       1.23     
virtualenv                    16.0.0   
wheel                         0.31.1   
wrapt                         1.10.11  
Check Information
=================
         Started: 2018-08-15 01:38:09.538781
        Finished: 2018-08-15 01:38:19.153013
      Time Taken: 9.61 seconds
       Formatter: grouped
        Profiles: /usr/src/databricks-cli/prospector.yaml, no_doc_warnings, strictness_high, strictness_veryhigh, no_member_warnings
      Strictness: from profile
  Libraries Used: 
       Tools Run: dodgy, mccabe, pep8, profile-validator, pyflakes, pylint, pyroma
  Messages Found: 0

Processing /usr/src/databricks-cli
Requirement already satisfied: click>=6.7 in /usr/local/lib/python2.7/site-packages (from databricks-cli==0.8.1) (6.7)
Requirement already satisfied: requests>=2.17.3 in /usr/local/lib/python2.7/site-packages (from databricks-cli==0.8.1) (2.19.1)
Requirement already satisfied: tabulate>=0.7.7 in /usr/local/lib/python2.7/site-packages (from databricks-cli==0.8.1) (0.8.2)
Requirement already satisfied: six>=1.10.0 in /usr/local/lib/python2.7/site-packages (from databricks-cli==0.8.1) (1.11.0)
Requirement already satisfied: configparser>=0.3.5 in /usr/local/lib/python2.7/site-packages (from databricks-cli==0.8.1) (3.5.0)
Requirement already satisfied: chardet<3.1.0,>=3.0.2 in /usr/local/lib/python2.7/site-packages (from requests>=2.17.3->databricks-cli==0.8.1) (3.0.4)
Requirement already satisfied: urllib3<1.24,>=1.21.1 in /usr/local/lib/python2.7/site-packages (from requests>=2.17.3->databricks-cli==0.8.1) (1.23)
Requirement already satisfied: certifi>=2017.4.17 in /usr/local/lib/python2.7/site-packages (from requests>=2.17.3->databricks-cli==0.8.1) (2018.8.13)
Requirement already satisfied: idna<2.8,>=2.5 in /usr/local/lib/python2.7/site-packages (from requests>=2.17.3->databricks-cli==0.8.1) (2.7)
Installing collected packages: databricks-cli
  Running setup.py install for databricks-cli: started
    Running setup.py install for databricks-cli: finished with status 'done'
Successfully installed databricks-cli-0.8.1
============================= test session starts ==============================
platform linux2 -- Python 2.7.15, pytest-3.7.1, py-1.5.4, pluggy-0.7.1
rootdir: /usr/src/databricks-cli, inifile:
plugins: cov-2.5.1
collected 147 items

tests/test_utils.py .....                                                [  3%]
tests/clusters/test_cli.py ........                                      [  8%]
tests/configure/test_cli.py ....                                         [ 11%]
tests/configure/test_config.py .....                                     [ 14%]
tests/configure/test_provider.py ..............                          [ 24%]
tests/dbfs/test_api.py ............                                      [ 32%]
tests/dbfs/test_dbfs_path.py ..........                                  [ 39%]
tests/groups/test_cli.py ...........                                     [ 46%]
tests/jobs/test_api.py .                                                 [ 47%]
tests/jobs/test_cli.py .......                                           [ 52%]
tests/libraries/test_cli.py .................                            [ 63%]
tests/runs/test_cli.py .....                                             [ 67%]
tests/sdk/test_api_client.py .                                           [ 68%]
tests/secrets/test_cli.py ...............                                [ 78%]
tests/stack/test_api.py ............                                     [ 86%]
tests/stack/test_cli.py ....                                             [ 89%]
tests/workspace/test_api.py ...............                              [ 99%]
tests/workspace/test_cli.py .                                            [100%]

========================== 147 passed in 1.09 seconds ==========================
Uninstalling pep8-naming-0.7.0:
  Successfully uninstalled pep8-naming-0.7.0
Uninstalling pytest-cov-2.5.1:
  Successfully uninstalled pytest-cov-2.5.1
Uninstalling pylint-1.9.3:
  Successfully uninstalled pylint-1.9.3
Uninstalling pytest-3.7.1:
  Successfully uninstalled pytest-3.7.1
Uninstalling decorator-4.3.0:
  Successfully uninstalled decorator-4.3.0
Uninstalling codecov-2.0.15:
  Successfully uninstalled codecov-2.0.15
Uninstalling rstcheck-3.3:
  Successfully uninstalled rstcheck-3.3
Uninstalling prospector-1.1.1:
  Successfully uninstalled prospector-1.1.1
Uninstalling mock-2.0.0:
  Successfully uninstalled mock-2.0.0
Removing intermediate container f8c93896b32e
 ---> 39710a86efa1
Step 5/5 : ENTRYPOINT [ "databricks" ]
 ---> Running in e1acd610410b
Removing intermediate container e1acd610410b
 ---> 1bab130a045d
Successfully built 1bab130a045d
Successfully tagged databricks-cli:latest
```

```
$ docker run databricks-cli
Usage: databricks [OPTIONS] COMMAND [ARGS]...

Options:
  -v, --version   0.8.1
  --debug         Debug Mode. Shows full stack trace on error.
  --profile TEXT  CLI connection profile to use. The default profile is
                  "DEFAULT".
  -h, --help      Show this message and exit.

Commands:
  clusters   Utility to interact with Databricks clusters.
  configure  Configures host and authentication info for the CLI.
  fs         Utility to interact with DBFS.
  groups     Utility to interact with Databricks groups.
  jobs       Utility to interact with jobs.
  libraries  Utility to interact with libraries.
  runs       Utility to interact with the jobs runs.
  secrets    Utility to interact with Databricks secret API.
  stack      Utility to deploy and download Databricks resource stacks.
  workspace  Utility to interact with the Databricks workspace.
```

**Comment by nfx** [comment]
Please upgrade to the latest version. It seems that the version you're referring is too old.
